wiki/Beatmap_Editor/Compose/en.md

```diff
@@ -120,60 +122,60 @@ Hold `Alt` to trigger Distance Snap. The value can range from x0.1 - x2.0.
-| Clap (`R`) | ("Select" on a Hit note) **Left-Click:** Use clap sound on this note | Add a clap sound addition to the selection. |
+| Whistle (`W`) | ("Select" on a hit note) **Left-Click:** Add a whistle sound on this note | Add a whistle sound addition to the selection. |
+| Finish (`E`) | ("Select" on a hit note) **Left-Click:** Add a finish sound on this note | Add a finish (cymbal) sound addition to the selection. |
+| Clap (`R`) | ("Select" on a hit note) **Left-Click:** Add a clap sound on this note | Add a clap sound addition to the selection. |
 
 #### Assist Tools
 
 | Button (Keyboard Shortcut) | Usage | Description |
 | -------------------------- | ----- | ----------- |
-| Grid Snap (`T`) | **`Shift` (Hold):** Temporary toggle. **`Ctrl` (Hold):** Temporary disable angle snapping. | Hit notes will be snapped to the grid by default. |
-| Distance Snap (`Y`) | **`Alt` (Hold):** Temporary toggle. Switch Beat Snap Divisor to Distance Snap **`Alt` + Mousewheel:** Adjust Distance Snap multiplier. | Distance between consecutive hit notes will be snapped based on their rhythmical difference. Best use while the timeline is paused. |
-| Lock Notes (`L`) | ("Select" on a Hit note) **Left-Click:** Lock this note | Lock selected note to current position and timestamp. |
+| Grid Snap (`T`) | **`Shift` (Hold):** Temporary toggle. **`Ctrl` (Hold):** Temporary disable grid snapping. | Hit notes will be snapped to the grid by default. |
+| Distance Snap (`Y`) | **`Alt` (Hold):** Temporary toggle. Switch Beat Snap Divisor to Distance Snap **`Alt` + Mousewheel:** Adjust Distance Snap multiplier. | Distance between consecutive hit notes will be snapped based on their rhythmical difference. This is best used while the timeline is paused. |
+| Lock Notes (`L`) | ("Select" on a hit note) **Left-Click:** Lock this note | Lock the selected note to the current position and timestamp. |
 
 ### Bottom (Song's Timeline)
 
 ![Song's Timeline](img/BE_HSAT.jpg "Song's Timeline")
 
-At the **bottom-left**, there is the **timestamp** in milliseconds (ms) and **song duration** in percent. The percentage may be switched to "intro" or "outro" if there is storyboarding before or after the music.
+At the **bottom-left**, there is the **timestamp** in milliseconds (ms) and **song duration** in percentage. The percentage will switch to "intro" or "outro" if there is storyboarding before or after the music.
 
-At the **bottom-centre**, there is a **timeline** with markings and the compulsory music player buttons. The `Test` button to the right of the timeline will save your beatmap then allows you to play test it, starting at the current timestamp.
+At the **bottom-centre**, there is a **timeline** with markings and the compulsory music player buttons. The `Test` button to the right of the timeline will save your beatmap then enter play test mode, starting from the current timestamp.
 
 #### Colour markers
```

Comment:
just post a picture with text inside of it here that's more straightforward than a table

Reply:
the entire section would do better by not being tables in my opinion